### PR TITLE
Add set position input to shutter forms

### PR DESCRIPTION
**Pull request by ChaosGramer**
Hello everyone, 

this PR contains a field on the form of the shutter which allows the user to set the target position of the shutter. 

Looking forward to hear from you. Regards, Jan

### CODE REVIEW

File: fs_src/index.html [hunk 1]
```diff
@@ -540,6 +540,13 @@ <h1 id="head">Window</h1>
           <label><span id="close_spinner"></span>Close</label>
         </button>
       </div>
+      <form class="form-control" id="set_tgt_pos_form">
+        <label>Set to Postion</label>
+        <input type="number" id="set_tgt_pos_input">
+        <button id="set_tgt_pos_button" type="submit">
```
Comment (timoschilling) [inline fs_src/index.html:546]: Remove the type=submit

File: fs_src/script.js [hunk 2]
```diff
@@ -489,6 +489,10 @@ function findOrAddContainer(cd) {
       el(c, "close_btn").onclick = function () {
         setComponentState(c, {tgt_pos: 0}, el(c, "close_spinner"));
       };
+      el(c, "set_tgt_pos_form").onsubmit = function (e) {
```
Comment (timoschilling) [inline fs_src/script.js:492]: Change onsubmit to onclick

File: fs_src/script.js [hunk 3]
```diff
@@ -489,6 +489,10 @@ function findOrAddContainer(cd) {
...
+        setComponentState(c, {tgt_pos: el(c, "set_tgt_pos_input").value}, el(c, "close_spinner"));
+        e.preventDefault();
```
Comment (timoschilling) [inline fs_src/script.js:494]: Can be removed

**Review by timoschilling — CHANGES_REQUESTED** [review]
Not tested but I expect that all requests together should work

File: fs_src/index.html (hunk 1 above)
Comment (ChaosGramer) [reply fs_src/index.html:546]: This way the enter key would not work that is why I choose to go for form + submit. Is this breaking anything?

Comment (timoschilling) [reply fs_src/index.html:546]: I see what you mean, but then this field will behave different then any other, so we should not add this feature to just one of them

Comment (ChaosGramer) [reply fs_src/index.html:546]: okay I see, I can adjust the other fields too, this will be straight forward.

Comment (timoschilling) [reply fs_src/index.html:546]: Maybe in a different PR

Comment (ChaosGramer) [reply fs_src/index.html:546]: okay I adjusted the code, please review again.

File: fs_src/script.js (hunk 2 above)
Comment (ChaosGramer) [reply fs_src/script.js]: done

File: fs_src/index.html (hunk 1 above)
Comment (timoschilling) [reply fs_src/index.html:546]: You missed to remove the type attr

Comment (ChaosGramer) [reply fs_src/index.html:546]: good catch

**Comment by ChaosGramer** [comment]
all comments solved I guess?

**Comment by timoschilling** [comment]
LGTM, but I’m on vacation and not able to test it

@rojer can you take a look

**Comment by rojer** [comment]
looks ok, thanks!

**Comment by rojer** [comment]
built and deployed the new fw, a couple comments:
 * please make the input smaller. it will only ever allow a 3 digit number, it doesn't need to be that big.
 * please change the label to "Position:" and center vertically

**Comment by rojer** [comment]
@ChaosGramer will you be able to send a followup or should I do it myself?

**Comment by ChaosGramer** [comment]
I tried already to fix it locally gimme the upcoming weekend looks like flex. box is not desirable for that purpose. 

Yeah I will send the follow up soon.

**Comment by rojer** [comment]
i needed to push 2.9.4 and reverted your commit temporarily, please resubmit with the changes.

**Pull request merged**